specs/chef/run_cookbook_manually.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Manually run a cookbook by name
+command: sudo chef client -o recipe['{{cookbook}}'] 
+tags:
+  - macos
+  - linux
+  - chef
+description: Use to fun a cookbook manually
```

Comment:
Is this supposed to be `run` instead of `fun?`

Reply:
oops